adoc-mode.el

```diff
@@ -2402,9 +2402,11 @@ Returns nil if there was no xref found."
                       (> saved-point (match-end 0)))))
       id)))
 
-(defun adoc-title-descriptor()
+(defun adoc-title-descriptor (&optional strict-match )
   "Returns title descriptor of title point is in.
 
+When STRICT-MATCH is t, 2 line title length must be within 2 chars of undertext
```

Comment:
I have to admit I don't understand what this means. :D I'm guessing that undertext is whatever is under the title. Btw, aren't the title separators different from those for other blocks?

Reply:
(I hate 2-line titles, as they require a lot of typing, and never use them myself)

Reply:
I will try to improve the description, how does "title underlining" sound.  Honestly didn't realize they existed until looking at this package.  Adoc-mode has support for fixing the number of repeated characters, so we need to have the non-strict mode for that functionality.  

++++ and ---- are two block starts that can be confused for title underlining.  It only works in practice because 4 character titles are not usual.  

Thinking about at this now, maybe just checking for > 4 repeated characters would be sufficient for differentiating blocks from section titles.  I'll take a look at this when I fix the description. 

BTW, it would be nice if the other failing tests were fixed so it's easier to see when changes break something.   Maybe disable them until they can be fixed?

Reply:
> BTW, it would be nice if the other failing tests were fixed so it's easier to see when changes break something. Maybe disable them until they can be fixed?

Agreed. I've opened an issue about fixing/disabling them a while ago, but I've been pretty short on time lately.